minicluster/src/main/java/tachyon/master/LocalTachyonMaster.java

```diff
@@ -84,6 +88,15 @@ private LocalTachyonMaster(final String tachyonHome, TachyonConf tachyonConf)
     mJournalFolder = mUnderFSCluster.getUnderFilesystemAddress() + "/journal";
 
     UnderFileSystemUtils.mkdirIfNotExists(mJournalFolder, tachyonConf);
+    String[] masterServiceNames = new String[] {
+        Constants.BLOCK_MASTER_SERVICE_NAME,
+        Constants.FILE_SYSTEM_MASTER_SERVICE_NAME,
+        Constants.RAW_TABLE_MASTER_SERVICE_NAME,
+    };
```

Comment:
Share this constant String array `masterServiceNames` with `minicluster/src/main/java/tachyon/master/LocalTachyonMaster.java` listed above.